tc/sgx/trusted_worker_manager/enclave/signup.edl

```diff
@@ -34,23 +34,59 @@ enclave {
 
         public tcf_err_t ecall_CreateSignupData(
             [in] const sgx_target_info_t* inTargetInfo,
-            [in, string] const char* inOriginatorPublicKeyHash,
             [out, size=inAllocatedPublicEnclaveDataSize] char* outPublicEnclaveData,
             size_t inAllocatedPublicEnclaveDataSize,
             [out, size=inAllocatedSealedEnclaveDataSize] uint8_t* outSealedEnclaveData,
             size_t inAllocatedSealedEnclaveDataSize,
             [out] sgx_report_t* outEnclaveReport
             );
 
+        public tcf_err_t ecall_CreateSignupDataKME(
```

Comment:
Create signup_kme.edl and signup_wpe.edl. And add required signup functions for KME and WPE in these files. Also later on when we create different enclave binaries, we need to create enclave_kme.edl and enclave_wpe.edl.

Reply:
Done